jetpack.php

```diff
@@ -154,11 +167,14 @@ function jetpack_admin_missing_autoloader() { ?>
 	return;
 }
 
-// Load all the packages.
+/**
+ * Load all the packages.
+ *
+ * We want to fail gracefully if `composer install` has not been executed yet, so we are including the autoloader.
+ * If `include` isn't able to load the autoloader, let's log the failure, pause Jetpack, and display a nice admin notice.
+ */
 $jetpack_autoloader = JETPACK__PLUGIN_DIR . '/vendor/autoload.php';
-if ( is_readable( $jetpack_autoloader ) ) {
```

Comment:
You meantioned "read one" in the PR title - does that imply that `is_readable` actually reads the file? I thought it only does some existence and permission checks?

Reply:
I meant read in terms of a disk read to see if the file is there.

Reply:
AFAIK since `is_readable` doesn't open the file, it's pretty fast - so I'd prefer keeping it.

Reply:
0238eab32222d21deda286d55b3551c2af48ee7b